pages/common/code.md

```diff
@@ -3,30 +3,34 @@
 > Visual Studio Code.
 > More information: <https://github.com/microsoft/vscode>.
 
-- Open VS Code:
+- Start the editor:
 
 `code`
 
-- Open the current directory in VS Code:
```

Comment:
This example is more useful and should be kept.

Reply:
I want make pages more general and demonstrate as much as possible command syntax. 😄 That's why I used:

```md
- Open specific files/directories:

`code {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```

`code` accepts not just one path but several ones.

Reply:
Personally, `code .` is the only command for code I run.

I'd imagine that not everyone knows that `.` is a valid, whilst im sure that most people will infer that you can enter another directory if you can do you current directory.